### ISSUE TITLE
Cannot run samples on Manjaro unix

### ISSUE THREAD

**Issue by tree786** [godlikepanos/anki-3d-engine#84]
Hallo fellow creations

I have compiled the repo on manjaro, but when I try and run `Sponza`, it gives me this error:
```
[E][VK  ][7f0db2c4a200] Couldn't find an async compute queue (/home/../Desktop/anki/AnKi/Gr/Vulkan/GrManagerImpl.cpp:516 initDevice)
[E][VK  ][7f0db2c4a200] Vulkan initialization failed (/home/../Desktop/anki/AnKi/Gr/Vulkan/GrManagerImpl.cpp:104 init)
[E][CORE][7f0db2c4a200] App initialization failed. Shutting down (/home/../Desktop/anki/AnKi/Core/App.cpp:315 init)
[E][N/A ][7f0db2c4a200] Error reported. Bye!! (/home/../Desktop/anki/Samples/Sponza/Main.cpp:40 main)

```
I have :

- GTX 760  card
- i7-4770 cpu (8 core)
- GCC 11.1.0
- CMake 3.21.1
- Kernel: 5.13.12-1 (manjaro) 

I can run other vulkan projects (like godot 4.0) with no problem, so not sure what issue is?

**Comment by godlikepanos** [comment]
Hi and thanks for the report. At the moment anki requires async compute to work and 760 doesn't seem to support it. I'm planning to fake async compute and I'll be implementing something soon. Stay tuned

**Comment by tree786** [comment]
Hallo panos brother,

Ok i see. Do you have idea when it will be complete? I like to start using the game soon

**Comment by tree786** [comment]
Is there a way i can "disable" async compute temporarily? Or is  it  glued to the code base and break rendering if i do so?

**Comment by godlikepanos** [comment]
Hi. It will take me a couple of days. Until I fix it you could checkout 98e3da123a0d1e75371a7894917b36257e8e6bb9 which doesn't require async compute.

**Comment by tree786** [comment]
ok i see, i think i should stick with that commit then, it compiles and runs fine on that commit
